kong/cmd/migrations.lua

```diff
@@ -24,7 +24,7 @@ The available commands are:
 
   list                              List executed migrations.
 
-  reset                             Reset the database.
+  reset                             Reset the database. This erases Kong data and doesn't reset migration steps.
```

Comment:
The `reset` command will erase all the data in Kong's database, and delete all the schemas.